include/aikido/perception/PoseEstimatorModule.hpp

```diff
@@ -54,6 +55,17 @@ class PoseEstimatorModule : public PerceptionModule
       ros::Duration timeout = ros::Duration(0.0),
       ros::Time timestamp = ros::Time(0.0)) override;
 
+  /// Getter for \c mObjInfo. \c mObjInfo is saving additional information
```

Comment:
How will this be used? It'd be really helpful to provide an example of what kind of JSON information would be sent, aside from the object ID.

In addition, does it make sense to return a YAML node here? What if we just return the string and delegate the JSON parsing to the client script? (I think this might also just be unclear to me since I'm not sure what information is being sent.)

Reply:
I also thought about this point. I just added DetectedObject class that wraps the YAML node and delegates a detected object so that we can pass more general information comparing to the skeleton in the aikido world. By using the JSON format, a third party perception algorithm can send any types of additional information to the application.